src/main/java/Deadline.java

```diff
@@ -0,0 +1,30 @@
+public class Deadline extends Task {
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        taskType = "D";
+    }
+
+    public static int addTask(Task[] taskList, int taskCount, String taskName, String taskBy) {
+        taskList[taskCount] = new Deadline(taskName, taskBy);
+
+        System.out.println(" Got it. I've added this task:\n" + "  [" + taskList[taskCount].taskType + "]["+ taskList[taskCount].getStatusIcon() + "] " + taskList[taskCount].description + " (" + taskList[taskCount].by + ")"  + "\n" + " Now you have " + (taskCount+1) + " tasks in the list.");
```

Comment:
This statement is quite long, maybe could use some refactors to make it shorter.